components/tab-panel/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { partial, noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { default as withInstanceId } from '../higher-order/with-instance-id';
+import { NavigableMenu } from '../navigable-container';
+
+const TabButton = ( { tabId, onClick, children, selected, ...rest } ) => (
+	<button role="tab"
+		tabIndex={ selected ? null : -1 }
+		aria-selected={ selected }
+		id={ tabId }
+		onClick={ onClick }
+		{ ...rest }
+	>
+		{ children }
+	</button>
+);
+
+class TabPanel extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.handleClick = this.handleClick.bind( this );
+		this.onNavigate = this.onNavigate.bind( this );
+
+		this.state = {
+			selected: this.props.tabs.length > 0 ? this.props.tabs[ 0 ].name : null,
+		};
+	}
+
+	handleClick( tabKey ) {
+		const { onSelect = noop } = this.props;
+		this.setState( {
+			selected: tabKey,
+		} );
+		onSelect( tabKey );
+	}
+
+	onNavigate( childIndex, child ) {
+		child.click();
+	}
+
+	render() {
+		const { selected } = this.state;
+		const {
+			activeClass = 'is-active',
+			className,
+			instanceId,
+			orientation = 'horizontal',
+			tabs,
+		} = this.props;
+
+		const selectedTab = tabs.find( ( { name } ) => name === selected );
```

Comment:
`Array#find` is not polyfilled and is not supported by IE11. This results in crashes in IE11 (e.g. opening inserter).